v0/destinations/posthog/data/PHAliasConfig.json

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "destKey": "properties.distinct_id",
+    "sourceKeys": [
+      "userId",
+      "context.traits.userId",
```

Comment:
Need to provide precedence to `traits.userId` and `traits.id`

Reply:
Fixed

Reply:
use the one in Generic Mapping .json
 "userId": [
    "userId",
    "traits.userId",
    "traits.id",
    "context.traits.userId",
    "context.traits.id",
    "anonymousId"
  ],

Reply:
Done